composer.json

```diff
@@ -55,7 +55,7 @@
         ]
     },
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.1",
         "ext-ctype": "*",
```

Comment:
You are aware that we have an LTS to support PHP &.4 until June 2023, and PHP 8.0 for another 18 months?